### DISCUSSION TITLE
Logout: customize redirect status

### DISCUSSION THREAD

**Discussion by jacargentina** [sergiodxa/remix-auth#131] [General]
In my case, I need the redirect to send a 205 HTTP status, so the browser forces a refresh on the target url.

Actually I copy/pasted the logout implant to simply pass it. Would be nice to have this on the API itself.

```js
import { ActionFunction, redirect } from 'remix';
import { destroySession, getSession } from '~/session.server';

export let action: ActionFunction = async ({ request }) => {
  let session = await getSession(request.headers.get('Cookie'));
  throw redirect('/', {
    status: 205,
    headers: {
      'Set-Cookie': await destroySession(session),
    },
  });
};

```

**Comment by sergiodxa** [comment]
Redirects are always 3xx status codes (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#redirection_messages)

If you want to force a refresh after the logout do `<Form action="/logout" method="post" reloadDocument>` or `<form action="/logout" method="post">` so the browser will reload the page to do the navigation to the logout page.
